docs/USAGE.md

```diff
@@ -30,14 +33,17 @@ The app follows the basic principles of the other openHAB UIs, like Basic UI, an
 ## Getting Started
 
 When first installed the app is in "Demo Mode".
-To connect it to your own openHAB server, first navigate to Settings and uncheck the "Demo Mode" option.
-Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to autodetect to your openHAB server if it is on the same network.
+To connect to your own openHAB server, navigate to the Settings menu and uncheck the "Demo Mode" option.
+Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to auto-detect your openHAB server if it is on the same network.
 
 You also have the option to manually set the server URL in the settings.
 
-**Local server settings:** Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
+**Local server settings:**
+Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
```

Comment:
Consider, "Please enter the base URL of your openHAB server, as you would..."

Reply:
IMHO it's correct without the comma.

Reply:
Yeah - it is a very small point.  It just gives the reader a hint about where to pause their thinking so that the sentence is a little easier to understand.  You could write it without the comma, and it would be gramatically correct.  But it is just like this preceding sentence.  "You could write it without the comma and it would be gramatically correct".  Think of it as sort of a grouping of sub-thoughts.  Or think of it as providing some very subtle guidance to the reader.  Not a big deal either way.

Reply:
Wouldn't you need a comma after 'browser' in this case?